understanding/22/accessible-authentication.html

```diff
@@ -44,7 +44,7 @@ <h2>Intent of Accessible Authentication</h2>
  
         <p>Remembering a password is a <a>cognitive function test</a>. Such tests are known to be problematic for many people with cognitive disabilities. Whether it is remembering random strings of characters, a pattern gesture to perform on a touch screen, or identifying which images include a particular object, cognitive function tests will exclude some people. When a cognitive function test is used, at least one other authentication method must be available which is not a cognitive function test.</p>
             
-        <p class="note">Web sites can employ username (or email) and password inputs as an authentication method if the inputs are properly marked up and do not block copy-paste functionality. Where the login form meets both <a href="identify-input-purpose">Success Criterion 1.3.5 Input Purpose</a> and  <a href="info-and-relationships">Success Criterion 1.3.1: Info and Relationships</a>, and does not block copy and paste, then browser features or password managers can save the user’s information and fill the credentials.</p>
+        <p class="note">Web sites can employ username (or email) and password inputs as an authentication method if the inputs are properly marked up and do not prevent paste functionality. Where the login form meets both <a href="identify-input-purpose">Success Criterion 1.3.5 Input Purpose</a> and  <a href="info-and-relationships">Success Criterion 1.3.1: Info and Relationships</a>, and does not prevent copying of usernames or pasting of credentials, then browser features or password managers can save the user’s information and fill the credentials.</p>
```

Comment:
https://github.com/w3c/wcag/issues/1855

Reply:
Hi @patrickhlauke, since my PR has not been pulled in yet, I think this is going to create some conflicts. Did you see my note on being concerned someone might interpret any copy-paste language as requiring an ability to copy an obscured password field string? Feel free to make your changes into my branch if you like, otherwise just as heads up to @alastc that there'll be a bit of clean up.

Reply:
> Did you see my note

where?

(and yes, in terms of (merge) conflicts, we can sort those out once one or the other get merged first)

Reply:
x-ref https://github.com/w3c/wcag/pull/1898/files#diff-7f63adbdc2bf700e35b33e2090452943b0168d306eaf9bdd9fe761bbcd608930R72

Reply:
minor: `and <a href="info-and-relationships"` stray extra space before `href`